2021q4/_index.adoc

```diff
@@ -29,6 +29,8 @@ toc::[]
 
 Entries from the various official and semi-official teams, as found in the link:../../administration/[Administration Page].
 
+include::{reports-path}/www.adoc[]
+
```

Comment:
@debdrup, I think you requested no changes to _index.adoc for your workflow and or sanity?

Reply:
Yes, I'd much prefer if _index.adoc is maintained by the status reports team.

Reply:
Sorry, fixed